src/Workspaces/Core/Desktop/Workspace/SQLite/Interop/OpenFlags.cs

```diff
@@ -23,8 +23,8 @@ internal enum OpenFlags
         // SQLITE_OPEN_SUBJOURNAL       = 0x00002000, /* VFS only */
         // SQLITE_OPEN_MASTER_JOURNAL   = 0x00004000, /* VFS only */
         // SQLITE_OPEN_NOMUTEX          = 0x00008000, /* Ok for sqlite3_open_v2() */
-        // SQLITE_OPEN_FULLMUTEX        = 0x00010000, /* Ok for sqlite3_open_v2() */
-        // SQLITE_OPEN_SHAREDCACHE      = 0x00020000, /* Ok for sqlite3_open_v2() */
+        SQLITE_OPEN_FULLMUTEX        = 0x00010000, /* Ok for sqlite3_open_v2() */
```

Comment:
sqlite seems weird that each connection basically open a db themselves and behave like 2 completely isolated connection.. anyway, shared cache seems mitigate that behavior.